src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -406,13 +411,23 @@ private static void checkPreconditions(Project project) {
                 "Gradle Consistent Versions doesn't currently work with configure-on-demand, please remove"
                         + " 'org.gradle.configureondemand' from your gradle.properties");
 
+        Multimap<String, Project> coordinateDuplicates = LinkedHashMultimap.create();
+        Set<Project> subprojectsLeft = new HashSet<>(project.getSubprojects());
```

Comment:
is concurrency important here? surely subprojects will end up getting evaluated on different threads??

Reply:
I don't think so, that would break the world. Oddly I can't find written evidence of this though :(